standalone-container/src/main/java/com/yahoo/container/standalone/StandaloneContainerApplication.java

```diff
@@ -340,4 +345,82 @@ private static void initializeContainerModel(ContainerModel containerModel, Conf
         return Optional.ofNullable(System.getProperty(name)); // for unit testing
     }
 
+    private static class StandaloneContainerModelContextProperties implements ModelContext.Properties {
+        private final CloudConfigInstallVariables cloudConfigInstallVariables;
+
+        StandaloneContainerModelContextProperties(CloudConfigInstallVariables cloudConfigInstallVariables) {
+            this.cloudConfigInstallVariables = cloudConfigInstallVariables;
+        }
+
+        @Override
+        public boolean multitenant() {
+            return cloudConfigInstallVariables.multiTenant().orElse(Boolean.FALSE);
+        }
+
+        @Override
+        public ApplicationId applicationId() {
+            return ApplicationId.defaultId();
+        }
+
+        @Override
+        public List<ConfigServerSpec> configServerSpecs() {
+            return null;
+        }
+
+        @Override
+        public HostName loadBalancerName() {
+            return null;
+        }
+
+        @Override
+        public URI ztsUrl() {
+            return null;
+        }
+
+        @Override
+        public String athenzDnsSuffix() {
+            return null;
```

Comment:
These should throw UnsupportedOperationException to ensure they're not called, and/or don't cause strange NPE later?

Reply:
Good point